app/controllers/mechanics_controller.rb

```diff
@@ -0,0 +1,13 @@
+class MechanicsController < ApplicationController
+  def show
+    @mechanic = Mechanic.find(params[:id])
+  end
+
+  def add_ride
```

Comment:
I see what you were trying to do here. You wanted to add to an existing mechanic, an existing ride. To make this more RESTful, we would want crate a join controller, something like:
```ruby
RideMechanicsController
```
where we would have a `create` action. We need to create a new association between the mechanic and the ride and that is how it is added to the mechanic. I will look at your form.